pym/bob/cmds/build.py

```diff
@@ -1365,13 +1365,23 @@ def doProject(argv, bobRoot):
         help="Do not build (bob dev) before generate project Files. RunTargets may not work")
     parser.add_argument('-b', dest="execute_buildonly", default=False, action='store_true',
         help="Do build only (bob dev -b) before generate project Files. No checkout")
+    parser.add_argument('--build', default=False, action='store_true',
+        help="Do path calculations as if 'bob build', not as 'bob dev'")
```

Comment:
Isn't that part of PR #170 already? It would anyway be an unrelated change for this PR.